diff -urN kshutdown-0.6.0.new/kshutdown/Makefile.in kshutdown-0.6.0/kshutdown/Makefile.in
--- kshutdown-0.6.0.new/kshutdown/Makefile.in	2005-05-13 16:37:16.000000000 +0200
+++ kshutdown-0.6.0/kshutdown/Makefile.in	2005-05-13 16:40:29.000000000 +0200
@@ -668,6 +668,7 @@
 #>- 	$(mkinstalldirs) $(kde_appsdir)/Utilities/
 #>+ 1
 	$(mkinstalldirs) $(DESTDIR)$(kde_appsdir)/Utilities/
+	$(mkinstalldirs) $(DESTDIR)$(kde_datadir)/kconf_update/
 #>- 	$(INSTALL_DATA) $(srcdir)/kshutdown.desktop $(kde_appsdir)/Utilities/kshutdown.desktop
 #>+ 1
 	$(INSTALL_DATA) $(srcdir)/kshutdown.desktop $(DESTDIR)$(kde_appsdir)/Utilities/kshutdown.desktop
